### PR TITLE
[PaddlePaddle Hackathon]  add paddle.vision.models.InceptionV3

### PR DESCRIPTION
**Pull request by SigureMo**
### PR types

New features

### PR changes

APIs

### Describe

- Task: https://github.com/PaddlePaddle/Paddle/issues/35998
- 添加 paddle.vision.models.InceptionV3

status: Merged

- [x] 支持预训练模型
- [x] 单元测试
- [x] 文档 https://github.com/PaddlePaddle/docs/pull/3921

### Performance

AI Studio 测试详情：<https://aistudio.baidu.com/aistudio/projectdetail/2420319?contributionType=1>
基准参考：<https://github.com/PaddlePaddle/PaddleClas/blob/release/2.2/docs/en/ImageNet_models_en.md>

|Model|Top-1|Top-5|
|-|-|-|
|InceptionV3|0.7906<span style="color: green;"><sub>(-0.0008)</sub></span>|0.9450<span style="color: green;"><sub>(-0.0009)</sub></span>|

括号中为以 PaddleClas 性能基准为参考的偏差值

### CODE REVIEW

**Review by LielinJiang — DISMISSED** [review]

File: python/paddle/vision/models/inceptionv3.py [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+
+        branch_pool = self.branch_pool(x)
+        branch_pool = self.branch_pool_conv(branch_pool)
+
+        x = paddle.concat(
+            [branch1x1, branch3x3, branch3x3dbl, branch_pool], axis=1)
+        return x
+
+
+class InceptionV3(nn.Layer):
```
Comment (TCChenlong) [inline python/paddle/vision/models/inceptionv3.py:433]: 英文文档的代码示例应与中文一致~

File: python/paddle/vision/models/inceptionv3.py [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+
+def inception_v3(pretrained=False, **kwargs):
+    """
+    InceptionV3 model from
+    `"Rethinking the Inception Architecture for Computer Vision" <https://arxiv.org/pdf/1512.00567.pdf>`_
+
+    Args:
+        pretrained (bool): If True, returns a model pre-trained on ImageNet
+    
+    Examples:
```
Comment (TCChenlong) [inline python/paddle/vision/models/inceptionv3.py:533]: 同上

Comment (SigureMo) [reply python/paddle/vision/models/inceptionv3.py:533]: 已修改～～～

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by LielinJiang — APPROVED** [review]

**Review by LielinJiang — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**